app/scripts/filters/resources.js

```diff
@@ -833,8 +841,14 @@ angular.module('openshiftConsole')
     };
   })
   .filter('jenkinsLogURL', function(annotationFilter) {
-    return function(build) {
-      return annotationFilter(build, 'jenkinsLogURL');
+    return function(build, asPlainText) {
+      var logURL = annotationFilter(build, 'jenkinsLogURL');
+      if (!logURL || asPlainText) {
+        return logURL;
+      }
+
+      // Link to the Jenkins console that follows the log instead of the raw log text.
+      return logURL.replace(/\/consoleText$/, '/console');
```

Comment:
I guess later when we want to have the log appear natively in the console we would need the /consoleText URL anyway